lib/tasks/lockout_moves.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+namespace :lockout_moves do
+  desc 'lockouts a move which containts a MoveLockut in its generic events'
+  task lockout_moves: :environment do
```

Comment:
I'd suggest putting this in the existing [`data_maintenance.rb`](https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/main/lib/tasks/data_maintenance.rake) group of Rake tasks.